indico/queries/model_groups/model_groups.py

```diff
@@ -395,3 +360,126 @@ def requests(self):
         yield _ModelGroupPredict(
             model_id=self.model_id, data=self.data, predict_options=self.predict_options
         )
+
+
+class AddModelGroupComponent(GraphQLRequest):
+    """
+    Adds a new model group component to a workflow, optionally with a customized questionnaire. Available on 5.0+ only.
```

Comment:
I think "component" is going to be a new term for people (honestly I'm not 100% sure what it means)

Reply:
Good point, I might see if there's a better term or phrasing, for now I'll specify it's a step of sorts